low-level/sensors/impl/generic/raid_integrity_data.py

```diff
@@ -103,15 +107,22 @@ def initialize(self, conf_reader, msgQlist, product):
         self._rack_id = self._conf_reader._get_value_with_default(
                                 self.SYSTEM_INFORMATION, COMMON_CONFIGS.get(self.SYSTEM_INFORMATION).get(self.RACK_ID), '001')
         self._node_id = self._conf_reader._get_value_with_default(
-                                self.SYSTEM_INFORMATION, COMMON_CONFIGS.get(self.SYSTEM_INFORMATION).get(self.NODE_ID), '001')  
+                                self.SYSTEM_INFORMATION, COMMON_CONFIGS.get(self.SYSTEM_INFORMATION).get(self.NODE_ID), '001')
         self._timestamp_file_path = self._conf_reader._get_value_with_default(
                                     self.RAIDIntegritySensor, self.TIMESTAMP_FILE_PATH_KEY, self.DEFAULT_TIMESTAMP_FILE_PATH)
         self._polling_interval = int(self._conf_reader._get_value_with_default(
```

Comment:
rename '_polling_interval ' to '_scan_frequency'

Reply:
even in sspl conf??

Reply:
yes

Reply:
here if we change it to `scan_frequency` instead of `polling_interval` in conf then we need to update sspl.sls.

Reply:
changed to `scan_frequency` only in raid_integrity Sensor, 
for `sspl.sls` and `sspl.conf`  => `polling_interval` is kept as it was.